.github/workflows/spellcheck.yml

```
@@ -32,6 +32,7 @@ jobs:
         id: spell
         with:
           dictionary: .github/components/dictionary.txt
+          files: "docs/* docs/*/* docs/*/*/*"
```

I think you want this to only fill in docs if it is a pull requets. I am not 100% sure this will do exactly what I would want, since I am not sure how `files: null` would be handled, bu tthe overall structure is the the one for a ternary operator in github actions. 
```suggestion
          files: ${{ github.event_name == 'pull_request' && "docs/* docs/*/* docs/*/*/*" || null }}
```